packages/suite/src/hooks/suite/useActionAbortable.ts

```diff
@@ -0,0 +1,10 @@
+import { useSelector } from '@suite-hooks/useSelector';
+import { versionUtils } from '@trezor/utils';
+
+export const useActionAbortable = () => {
```

Comment:
please, change it to a selector 🙏

Reply:
22c2636cf5aaacf691e19ca0a56d713d59683f34